### ISSUE TITLE
Community volunteer open data census reviewer 

### ISSUE THREAD

**Issue by MireyaGalarza** [open-austin/project-ideas#67]
### What problem are we trying to solve?

Need a community volunteer for Open Data Census 
### Who will benefit (directly and indirectly) from this project?

Open data community
### Links to any research/data available/articles

Links here.http://us-city.census.okfn.org/year/2016
### What are the next steps (validation, research, coding, design)?

Create an account http://us-city.census.okfn.org/ and volunteer to be set up as a reviewer.
### What help is needed at this time?

We ask that reviewers be community members who are not city hall employees. See more below from our FAQ page:

**How do I become my city’s Open Data Census Librarian?**
Open Data Census Librarians are the reviewers and point persons for the Census assessment in their community. They are responsible for filling out a profile page, becoming familiar with this FAQ and the Data Explainers, and reviewing open data periodically.

**Open Data Census Librarian Basic Responsibilities**
By adopting a US City Census page, you agree to do the following:
Become familiar enough with the US City Open Data FAQ and Data Explainers so that you can serve as a front-line resource on these materials to your community. 

Periodically review community datasets to keep your community up-to-date, i.e. review new datasets monthly (and/or after coordinated national hackathons), and all datasets annually. 

Review datasets assessed by your fellow community members, checking for: missing or dead links, and mis-assessed data availability criteria (including is the data: available online, publicly accessible, cost-free, machine-readable, available in bulk, license-free, and up-to-date.)

**Comment by luqmaan** [comment]
@MireyaGalarza How do I become a reviewer? I've created an account and I'm not sure what to do next. I tried to propose some changes to Austin's transit dataset (the old link is broken), but got this error `Luq, You do not have permissions to review this submission.`

**Comment by MireyaGalarza** [comment]
@luqmaan   Please send an email  request for "reviewer" access to 
 [mailto:usopendatacensus@gmail.com] .  
Thank you

**Comment by amaliebarras** [comment]
Update! This project needs reviewers and contributions for the 2017 census too!

**Comment by tracypholmes** [comment]
Will give it a looksie before I commit. This appears easy enough for me at first glance. What number of reviewers is needed?

**Comment by amaliebarras** [comment]
great! @MireyaGalarza can you advise on number of reviewers?

**Comment by MireyaGalarza** [comment]
Five volunteers would be a good number.  There are 19 sets to review and most of them are on a scheduled updates since last year.

**Comment by MireyaGalarza** [comment]
oops! did not mean to close.

**Comment by tracypholmes** [comment]
Thanks - will make sure to sign up to volunteer as a reviewer! Started a new job so had to push some stuff to the back for a minute.

**Comment by werdnanoslen** [comment]
Hi @MireyaGalarza what's the status on the census volunteer list so far?

**Comment by werdnanoslen** [comment]
Thanks for the update @MireyaGalarza! We'd love to have you at our next hack night to pitch this to our members. Most of the folks in this thread aren't as active anymore, so feel free to join us over some tacos next time: http://meetu.ps/c/1FjlF/2Nl9B/d

**Comment by MireyaGalarza** [comment]
@werdnanoslen I can not do the review.  :(  I submitted the changes.

**Comment by werdnanoslen** [comment]
Oh right, I just mean since our meetups are the best place to ask for help, it'd be great if you could join us to pitch the overall concept of what census community reviewers do.

**Comment by werdnanoslen** [comment]
Hi @MireyaGalarza, @VictoriaODell and I were just trying to contribute to this, but we ran into some issues. Could you please help us out with these issues? https://github.com/okfn/opendatasurvey/issues

**Comment by VictoriaODell** [comment]
@werdnanoslen, @MireyaGalarza is at CTM and doesn't have much control over the issues in the website repo. 

I Spoke with @MireyaGalarza recently though and CTM has a sprint set to end for October 11th where they were hoping to get 2017 all done. I checked in on the year though: 
http://us-city.census.okfn.org/year/2017

and there is a lot more work to be done to get it back up to the top. 

<img width="1392" alt="screen shot 2017-09-30 at 12 05 13 pm" src="https://user-images.githubusercontent.com/10767430/31047853-ab3258b8-a5d7-11e7-9cf0-dcabf42cf770.png">

**Comment by VictoriaODell** [comment]
got in contact with someone. Here is the response

> I have added victoria.d.odell@gmail.com as a reviewer for Austin. If you scroll to the bottom of the Census page, in the lower-right corner there's a place where you can log in. When a new dataset submission comes in for Austin, you will see a "1" icon on the main Census page (look at other cities to see what I mean). Alternatively, you click on "Austin, TX" you'll see any pending datasets. You'll be able to review and publish any pending submissions.
> 
> For the other reviewer requests, is there any chance it might be possible to send a list of them all at once? It's a lot easier for me that way. :)
> 

> Re: "Also are you one of the core maintainers for the open data census?" Yes, sort of. Responsibility for managing the Open Data Census has three parts:
> **The technical side:** Open Knowledge International runs the platform that the Census runs on. This is just one instance of many open-data censuses around the world, and they all run on the same platform.
> **The administrative side:** We at Sunlight do that — "we" currently being me. This involves adding new cities and reviewers. (And theoretically, changing what the datasets are, although we haven't done that for longitudinal-comparison reasons.)
> **The submission-review side:** This is done by the reviewers in individual cities. Many of the reviewers are involved in CfA brigades, although there are plenty who are not. (Note that although Code for America's logo is at the top of the Census page alongside OKI and Sunlight's, CfA national HQ is not involved in the Census, although I think they maybe once were.)
>
>  Let me know if you have any other questions!
> 

@werdnanoslen are you interested in being a reviewer too?

**Comment by werdnanoslen** [comment]
You can add me as long as it's not a commitment, I think we have enough on our plate, but the more we know about this the easier it'll be to onboard other members.

**Comment by MireyaGalarza** [comment]
@VictoriaODell and @werdnanoslen  thank you so much for looking into it.

**Comment by actuallyjenn** [comment]
For the Civic Hack Night on Oct 11th, I reviewed, updated, and submitted the Parcels dataset - http://us-city.census.okfn.org/submission/240d670f-b895-4949-b937-b6a41edddfbc. It needs to be reviewed.

After some digging, @VictoriaODell and I realized that it'd be best if we could capture some documentation on what needs to be done and what the process should look like. I've started to do that and the draft can be found here: https://docs.google.com/presentation/d/1DhA7eIGYUawt2WEPFhP8_cSwRJC349jTtEWbRnWke0k/edit?usp=sharing.

I'm going to keep updating the document and also request access to become a reviewer to review the other 7 submissions that are pending.

**Comment by VictoriaODell** [comment]
This looks great @actuallyjenn! I could see us doing a Medium post about this too with the way you laid out the content. What is your best email so I can add you as a reviewer? 


@MireyaGalarza Might be good for us to sync some. It seem there are still some data sets that need to be submitted.

**Comment by MireyaGalarza** [comment]
@actuallyjenn thank you so much for looking into it and putting documentation together.   @VictoriaODell I will go and submit the rest.  Have you and other members received REVIEWER access?

**Comment by MireyaGalarza** [comment]
@VictoriaODell I submitted the rest of the revisions for 2017 except for public buildings.  There seems to be a problem with the file in Socrata. Most Of the files that were reviewed in 2016 are still valid because they are on Socrata and have scheduled updates.

**Comment by VictoriaODell** [comment]
@MireyaGalarza I currently have access. Wrangling a couple of other members. 

@actuallyjenn are you still interested in helping? can I get your email address to get you added?

**Comment by actuallyjenn** [comment]
@VictoriaODell yeah! add me as a reviewer and I can review the datasets: jnoinaj@gmail.com. I can't make it this Saturday but planning on finishing up the documentation in my free time and can also write a Medium post!

**Comment by VictoriaODell** [comment]
Sending your email off today  @actuallyjenn. You should receive a confirmation some time this week.

@wilsaj and @luqmaan I see ya'll were reviewers in the past. Would ya'll have any interest in reviewing this year?

**Comment by fruitschinposamurai** [comment]
@VictoriaODell I would be interested in reviewing- I'm assuming the same thing would happen for next year's data now?

**Comment by actuallyjenn** [comment]
I went in last week and reviewed / approved the Austin dataset. http://us-city.census.okfn.org/year/2017

I just noticed someone else also did the same? This is awesome that for 2017, we managed to get Austin to the #1 spot. I am looking forward to the new platform for 2018.

![image](https://user-images.githubusercontent.com/6501138/34617067-c5ea8bde-f1ff-11e7-80d0-a137397abf0e.png)

**Comment by VictoriaODell** [comment]
oh, wow! Great job folks. And yes @fruitschinposamurai there should be a review next year if you provide your email address I can get you added to the list. As mentioned by @actuallyjenn they are doing some changes for 2018, [details in this blog](https://sunlightfoundation.com/2017/11/20/changes-are-coming-to-the-u-s-city-open-data-census/).

**Comment by fruitschinposamurai** [comment]
Sure, @VictoriaODell my email is akhiloar@gmail.com.

**Comment by VictoriaODell** [comment]
We did it for 2018 so placing this in our project backlog milestone

**Comment by MireyaGalarza** [comment]
New site for 2018 is ready to be updated.  http://us-cities.survey.okfn.org/  I just added Budget.
